docs/release-notes/1.10.2.md

```diff
@@ -21,4 +21,5 @@
 ```
 
 * `sparse_mean_variance_axis` now uses all cores for the calculations {pr}`3015` {smaller}`S Dicks`
+* `pp.highly_variable_genes` with `flavor=seurat_v3` now uses all cores for clipping {pr}`3017` {smaller}`S Dicks`
```

Comment:
This isn't parallel, right?

Reply:
No it's not I'll update the release note